cpp/include/cudf/utilities/span.hpp

```diff
@@ -120,6 +120,11 @@ struct is_host_span_supported_container<  //
   thrust::host_vector<T, Alloc>> : std::true_type {
 };
 
+template <typename T, typename Alloc>
+struct is_host_span_supported_container<  //
```

Comment:
```suggestion
struct is_host_span_supported_container<
```

Reply:
This is exactly the same as the two existing definitions. I did not add this.